include/vinecopulib/bicop/implementation/bb6.ipp

```diff
@@ -23,21 +21,21 @@ inline Bb6Bicop::Bb6Bicop()
 
 inline double Bb6Bicop::generator(const double &u)
 {
-    double res = boost::math::log1p(-std::pow(1 - u, parameters_(0)));
+    double res = tools_stl::log1p(-std::pow(1 - u, parameters_(0)));
     return std::pow((-1) * res, parameters_(1));
 }
 
 inline double Bb6Bicop::generator_inv(const double &u)
 {
-    double res = boost::math::expm1(-std::pow(u, 1 / parameters_(1)));
+    double res = std::expm1(-std::pow(u, 1 / parameters_(1)));
```

Comment:
std::expm1 is better than boost::math::expm1?

Reply:
Same question for stp::log1p btw.

Reply:
I'd always prefer using STL versions when they exist, but no strong opinion on that.

Reply:
Ok, I didn't even know that STL had such functions (just noticed that they were C++11, which probably explains why I didn't find them initially).

I'm all good once the tests pass then.